cmd/dispatcher/main.go

```diff
@@ -74,6 +75,11 @@ func main() {
 		logger.Fatalf("Failed to build coordination client: %s", err.Error())
 	}
 
+	redisPW, err := ioutil.ReadFile("redis.conf")
```

Comment:
The file path has to be taken from a flag.

Reply:
see https://github.com/vega-project/ccb-operator/blob/master/cmd/dispatcher/main.go#L24-L38